com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/HDRP/HDSampleBufferNode.cs

```diff
@@ -0,0 +1,144 @@
+                        //I don't see the sampler param getting used anywhere.  Is it needed?
+                        new ParameterDescriptor("Sampler", TYPE.SamplerState, Usage.In),
+                        new ParameterDescriptor("Out", TYPE.Float, Usage.Out)
+                    },
+                    new string[]
+                    {
+                        "\"Packages/com.unity.render-pipelines.high-definition/Runtime/Material/NormalBuffer.hlsl\"",
+                        "\"Packages/com.unity.render-pipelines.high-definition/Runtime/Material/Builtin/BuiltinData.hlsl\""
+                    }
+                ),
+                new (
+                    "MotionVectors",
+@"   uint2 pixelCoords = uint2(UV * _ScreenSize.xy);
+   float4 motionVecBufferSample = LOAD_TEXTURE2D_X_LOD(_CameraMotionVectorsTexture, pixelCoords, 0);
+   float2 motionVec;
+   DecodeMotionVector(motionVecBufferSample, motionVec);
+   Out = motionVec;",
+                    new ParameterDescriptor[]
+                    {
+                        new ParameterDescriptor("UV", TYPE.Vec2, Usage.In, REF.ScreenPosition_Default),
+                        //I don't see the sampler param getting used anywhere.  Is it needed?
+                        new ParameterDescriptor("Sampler", TYPE.SamplerState, Usage.In),
+                        new ParameterDescriptor("Out", TYPE.Vec2, Usage.Out)
+                    },
+                    new string[]
+                    {
+                        "\"Packages/com.unity.render-pipelines.high-definition/Runtime/Material/NormalBuffer.hlsl\"",
+                        "\"Packages/com.unity.render-pipelines.high-definition/Runtime/Material/Builtin/BuiltinData.hlsl\""
+                    }
+                ),
+                new (
+                    "IsSky",
+@"   Out = IsSky(UV) ? 1 : 0;",
+                    new ParameterDescriptor[]
+                    {
+                        new ParameterDescriptor("UV", TYPE.Vec2, Usage.In, REF.ScreenPosition_Default),
+                        //I don't see the sampler param getting used anywhere.  Is it needed?
+                        new ParameterDescriptor("Sampler", TYPE.SamplerState, Usage.In),
+                        new ParameterDescriptor("Out", TYPE.Float, Usage.Out)
+                    },
+                    new string[]
+                    {
+                        "\"Packages/com.unity.render-pipelines.high-definition/Runtime/Material/NormalBuffer.hlsl\"",
+                        "\"Packages/com.unity.render-pipelines.high-definition/Runtime/Material/Builtin/BuiltinData.hlsl\""
+                    }
+                ),
+                new (
+                    "PostProcessInput",
+//The first two lines of body code live outside the function in SG1.  Does that matter?
+//In SG1, _CustomPostProcessInput is a dynamic variable. How do we do that?
+@"   TEXTURE2D_X(_CustomPostProcessInput);
+   SAMPLER(sampler_CustomPostProcessInput);
```

Comment:
Not sure about this one.

Reply:
for the `_CustomPostProcessInput` maybe we can have it as an HD special Ref?

Reply:
I made a note of this in the "GTF Miragtion Nodes Need Follow-ups" doc.